### DISCUSSION TITLE
プロジェクトの目標

### DISCUSSION THREAD

**Discussion by saito-simaru** [EyeTracker-123/UnityProject#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saito-simaru** [comment]
目標案
・視点移動
アイトラッカーの座標平均を取得して移動
・見た対象が変化する
見られたものに何にかしら変化を与える。
　　　・オブジェクトを飛ばして、対象物とのぶつかる面積

**Comment by OzuAkira** [comment]
# アイデア

3人称視点のゲームでプレイヤーを操作し、
アイトラッカーのデータは、神視点として運用

**Comment by OzuAkira** [comment]
# アイデア
まばたき　or 　目をつむっている間は音がきこえるゲーム

**Comment by saito-simaru** [comment]
以下ゲームの流れ

**Comment by saito-simaru** [comment]
ギミック

必ず鬼に追われる場所で目を瞑ることでしか見つからない打開の糸口を配置
